lib/apexcharts/options/legend.rb

```diff
@@ -1,24 +1,29 @@
 module ApexCharts
   class LegendOptions < ::SmartKv
     optional *%i[
-      containerMargin
+      show
+      showForSingleSeries
+      showForNullSeries
+      showForZeroSeries
       floating
-      fontFamily
+      inverseOrder
```

Comment:
`inverseOrder` on Legend is not released yet.
source: https://github.com/apexcharts/apexcharts.js/blob/512148c1462a4c8c9d3beca4988a6cc36af4b99b/types/apexcharts.d.ts